app/lib/templates.dart

```diff
@@ -300,14 +302,13 @@ class TemplateService {
   }
 
   /// Renders the `views/layout.mustache` template.
-  String renderLayoutPage(
-    String title,
-    String contentString, {
-    PackageVersion packageVersion,
-    Map<String, String> pageMapAttributes,
-    String faviconUrl,
-    String searchQuery,
-  }) {
+  String renderLayoutPage(String title, String contentString,
+      {PackageVersion packageVersion,
+      Map<String, String> pageMapAttributes,
+      String faviconUrl,
+      String searchQuery,
+      bool includeSurvey: true}) {
+    includeSurvey ??= true;
```

Comment:
It makes little sense to have `bool includeSurvey: true` if you check again for `null` and set it to `true`.

Either the caller is responsible for never passing `includeSuvey: null` or you remove the `: true`.